ofrak_components/setup.py

```diff
@@ -42,6 +42,7 @@ def run(self):
         "pycdlib==1.12.0",
         "python-magic",
         "synthol~=0.1.1",
+        "reedsolo",
```

Comment:
You should probably pin the version of this dependency so an upgraded version doesn't break the tests.